src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/InputIndentationCorrect.java

```diff
@@ -99,5 +102,9 @@ public String trimTrailingFrom(CharSequence sequence) {
         };
 
     private static final String ZEROES = "0\u0660\u06f0" + "\u0c66\u0ce6" + "\u1c40\u1c50";
+    // 3 violations above:
+    //  'Unicode escape(s) usage should be avoided.'
+    //  'Unicode escape(s) usage should be avoided.'
+    //  'Unicode escape(s) usage should be avoided.'
```

Comment:
How these violations are related to the current section?

Reply:
My bad, they aren't related. I accidentally kept them. Should I remove the whole variables or just fix the violations? I don't think the variable is used to cover any edge cases.

Reply:
there is no Indentation related content just make 
`private static final String ZEROES = "" + "";`

Reply:
done